lib/game.rb

```diff
@@ -0,0 +1,70 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+
+  def initialize(player_1=Player::Human.new("X"), player_2=Player::Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    board.turn_count % 2 == 0 ? self.player_1 : self.player_2
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def won?
+    combo_found = nil
+    combo_found = WIN_COMBINATIONS.find do |combo|
+      combo.all?{|space| board.cells[space] == "X"} || combo.all?{|space| board.cells[space] == "O"}
+    end
+    combo_found
+  end
+
+  def draw?
+    !won? && board.full?
+  end
+
+  def winner
+    won? ? board.cells[won?[0]] : nil
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.

If you prefer the single-line approach, you could also use
```
board.cells[won[0]] if won = won?
```
which will still return `nil` if the assignment fails.